_posts/2017/2017-08-17-tracing-the-suspect.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "Tracing the suspect - a microservices murder mystery"
+date: "2017-08-17 09:30:00"
+image: blog-header/tracing-the-suspect.jpg
```

Comment:
I like https://pixabay.com/en/night-luna-sky-full-moon-clouds-578091/ and https://unsplash.com/photos/9PPvnEd92xE. You make the final decision. Or just stick to the gun :-)